src/test/kotlin/com/tschuchort/compiletesting/KotlinCompilationTests.kt

```diff
@@ -645,6 +669,12 @@ class KotlinCompilationTests {
 			return fail<Nothing>("Class $className could not be loaded")
 		}
 	}
+
+	private fun classpathOf(dependency: String): File {
+		val regex = Regex(".*$dependency\\.jar")
+		val classGraph = ClassGraph().whitelistJars()
```

Comment:
Why are you using `whitelistJars()` here? If I read the [documentation](https://static.javadoc.io/io.github.classgraph/classgraph/4.8.5/io/github/classgraph/ClassGraph.html#whitelistJars(java.lang.String...)) correctly, this method should return an empty `ClassGraph` if no arguments are given.

Reply:
Oh, sorry, fixed!